src/App.css

```diff
@@ -1,25 +1,75 @@
 body {
-  background-color: #121212;
-  color: #cacaca;
+  display: flex;
+  flex-direction: column;
+  justify-content: top;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  transition: background 0.1s linear;
+  color: #fff;
+  background-color: #292c35;
 }
 
-.square {
-  width: 150px;
-  height: 150px;
-  border-color: #121212;
-  border-width: 5px;
-  font-size: 30px;
-  border-radius: 20%;
-  background: white;
+.head {
+  text-align: center;
+  opacity: 70%;
 }
 
 .content {
   display: flex;
-  justify-content: center;
-  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: top;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  transition: background 0.1s linear;
+}
+
+.light-and-switch {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: auto;
+}
+
+.checkbox {
+  opacity: 0;
+  position: absolute;
 }
 
-.top-buttons {
+.label {
+  background-color: #111;
   display: flex;
-  justify-content: center;
+  align-items: center;
+  justify-content: space-between;
+  border-radius: 7px;
+  position: relative;
+  margin-top: 60px;
+  padding: 7px;
+  height: 26px;
+  width: 50px;
+  transform: rotate(-90deg) scale(2);
+}
+
+.switch {
```

Comment:
For better clarity consider naming conventions to be more specific, maybe think about renaming switch to LightSwitch-switch if everything in css will be stored in one file.